.github/workflows/release.yml

```diff
@@ -118,38 +117,12 @@ jobs:
           echo "Publish docs to Confluence: $PUBLISH_DOCS"
           echo "Append notes to Confluence: $APPEND_NOTES"
```

Comment:
- The `Append Fix Notes to Confluence` step processes `new_fix_notes` for Confluence formatting but also includes a step to combine it with existing content. Make sure the combination doesn't introduce formatting conflicts.